docs/theory/kinetic_model.rst

```diff
@@ -0,0 +1,118 @@
+
+.. math::
+    \frac{dA}{dt} = v_1(A,\theta_1) - v_2(A, \theta_2)
+
+If we know the functions for :math:`v_1` and :math:`v_2` we can determine the concentration of :math:`A` over time, given an initial state :math:`A_0` 
+and :math:`theta_i`.
+
+.. math::
+    A_t = \int_0^t v_1(A,\theta_1) - v_2(A, \theta_2) dt
+
+.. figure:: Metabolite_A.png
+    :width: 400
+
+A metabolic network is formed by connecting metabolites through reactions, thus creating a tightly coupled network. 
+For example, if :math:`v_2` leads to a metabolite :math:`B`, the concentration of :math:`B` is dependent on :math:`A`. The resulting system of ODEs 
+must be solved simultaneously.
+
+The network can be represented with a stoichiometrix matrix (S) in which each row represents a metabolite, each 
+column a reaction and each entry represents the corresponding stoichiometric coefficient for the metabolite in the 
+reaction. The following figure is a three reaction network which is represented in a stoichiometric matrix
+
+.. figure:: S_matrix.png
+    :width: 400
+
+Using the stoichiometric matrix, we can express the system of ODEs in concise matrix form:
+
+.. math::
+    \frac{dC_i}{dt} &= \Sigma_j S_{ij} v_j \\
+    \frac{dC}{dt} &= Sv
+
+where :math:`S_{ij}` is the stoichiometric coefficient of metabolite :math:`i` in reaction :math:`j` and :math:`v_j` is the flux for reaction :math:`j`.
+
+Steady State Assumption
+=======================
+
+Maud evaluates the metabolism to a psuedo-steady-state such that:
+
+.. math::
+    \frac{dC_i}{dt} &= 0 \\
+    \text{or,} \\
+    Sv &= 0
+
+The following assumptions are made about the system:
+
+1. Enzyme concentrations are constant over time, and,
+2. The flows In and Out of a metabolite due to reactions are much higher than the dilution rate due to growth.
+
+Assumption 2. can be relaxed by accounting for the growth rate, however, we leave this to the user to implement.
+
+Flux vector representation
+==========================
+
+In a kinetic model, each flux is expressed as a rate law:
+
+.. math::
+    v_j = f(E,C,k, \Delta G^0)
+
+
+where: 
+
+* :math:`j` is the reaction :math:`j`;
+* :math:`E` is the enzyme concentration;
+* :math:`C` are the metabolite concentrations;
+* :math:`k` are the kinetic parameters;
+* :math:`\Delta G^0` is the standard Gibbs energy of reaction
+
+The rate laws can incorporate allosteric effectors, competitive inhibition, reaction mechanisms, 
+and varied protein concentrations. Common rate laws include: mass action kinetics where the rate 
+is a scaled-product of the reactant concentrations, or Michaelis-Menten kinetics which reproduce enzyme 
+saturation behavior.
+
+The current default mechanism is the :doc:`Modular Rate Law <../theory/enzyme_kinetics>`, which is augmented
+using the Generalised Monod-Wyman-Changeux formalism for allostery. Additional rate laws can be easily implemented 
+by the user.
+
+
+Out of Scope
+============
```

Comment:
It's probably worth putting a quick introduction sentence here, something like "Maud doesn't attempt to capture the following kinetic phenomena."